XCUITests/DragAndDropTests.swift

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+let firstTab = ["url": "www.youtube.com", "tabName": "Home - YouTube"]
+let secondTab = ["url": "www.twitter.com", "tabName": "Twitter"]
+
+class DragAndDropTests: BaseTestCase {
+
+    override func tearDown() {
+        XCUIDevice.shared().orientation = UIDeviceOrientation.portrait
+        super.tearDown()
+    }
+
+    private func openTwoWebsites() {
+        // Open two tabs
+        navigator.openURL(firstTab["url"]!)
+        navigator.goto(TabTray)
+        navigator.openURL(secondTab["url"]!)
+        waitUntilPageLoad()
+    }
+
+    private func checkTabsOrderBeforeDragAndDrop() {
+        // Check the order of the tabs before drag and drop
+        let firstTabCell = app.collectionViews.cells.element(boundBy: 0).label
+        let secondTabCell = app.collectionViews.cells.element(boundBy: 1).label
+
+        XCTAssertEqual(firstTabCell, firstTab["tabName"]!, "first tab before is not correct")
+        XCTAssertEqual(secondTabCell, secondTab["tabName"]!, "second tab before is not correct")
+    }
+
+    private func checkTabsOrderAfterDragAndDropRearrange() {
```

Comment:
how about if we just create one method, checkTabsOrder(bool: before) and do

if (before)
...do before check
else
... do after check

this would be easy to read and reduce duplicated code?